apps/cnspec/cmd/scan.go

```diff
@@ -63,6 +63,7 @@ func init() {
 	scanCmd.Flags().MarkHidden("category")
 	scanCmd.Flags().Int("score-threshold", 0, "If any score falls below the threshold, exit 1.")
 	scanCmd.Flags().Bool("share", false, "create a web-based private reports when cnspec is unauthenticated. Defaults to false.")
+	scanCmd.Flags().String("report-dest-url", "", "Set the URL to send the asset report to. Currently only supports AWS SQS topic URLs.")
```

Comment:
What about `output-target`? I find it weird to pass an SQS url to something called `output-file`

Also, a normal SQS url looks like this:
```
https://sqs.us-east-1.amazonaws.com/177715257436/MyQueue
```
I think we should stick that, instead of asking them to provide it with the `awssqs://` prefix (that seems to be specific to the gocloud dev lib we use). If we want that, we should make it clear in the doc with an example